x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -616,11 +616,7 @@ void decodeToken(String token, ActionListener<UserToken> listener) {
             }
         } catch (Exception e) {
             // could happen with a token that is not ours
-            if (logger.isDebugEnabled()) {
-                logger.debug("built in token service unable to decode token", e);
-            } else {
-                logger.warn("built in token service unable to decode token");
```

Comment:
Surprised that debug cancelled out the higher-severity warning here. Was the intention originally to avoid the exception making it into the warning? (just out of curiosity!)